salt_stack/salt/top.sls

```diff
@@ -27,6 +27,7 @@ base:
         - git
         - ingest-client.ingest
         - chrony
+        - open-files.increase-open-files
```

Comment:
Is this an existing formula or did you forget to add it to the branch?

Reply:
It was an existing Formula to fix a problem encountered on the activities server